app/connector/aws-ddb/src/main/resources/META-INF/syndesis/connector/aws-ddb.json

```diff
@@ -0,0 +1,235 @@
+{
+  "actions": [
+    {
+      "actionType": "connector",
+      "description": "Query items from an AWS Dynamo DB",
+      "descriptor": {
+        "componentScheme": "aws-ddb",
+        "connectorCustomizers": [
+          "io.syndesis.connector.aws.ddb.customizer.DDBConnectorCustomizerQuery"
+        ],
+        "inputDataShape": {
+          "kind": "any"
+        },
+        "outputDataShape": {
+          "kind": "any"
```

Comment:
For a future development iteration, make sure to properly convert to the class provided by the component or to a json. Same apply to any kind of input/output data shape.

Reply:
Yes, I am going to work next on extracting the table metadata once I finish with this PR.